src/mlpack/methods/hoeffding_trees/hoeffding_tree_main.cpp

```diff
@@ -182,7 +183,19 @@ static void mlpackMain()
       Log::Info << datasetInfo.NumMappings(i) << " mappings in dimension "
           << i << "." << endl;
 
-    labels = CLI::GetParam<arma::Row<size_t>>("labels");
+    if (CLI::HasParam("labels"))
+    {
+      labels = std::move(CLI::GetParam<arma::Row<size_t>>("labels"));
+    }
+    else
+    {
+      // Extract the labels as the last
```

Comment:
Pedantic spelling/style issue; can you check the punctuation. Also, should we use ``of the training set`` here? Let me know what you think.

Reply:
Sure, I'll make the necessary edit.